make/upgrade

```diff
@@ -38,15 +37,16 @@ source "${NETWORK_ENV}"
 
 UAA_CA_CERT="$(get_secret "${UAA_NAMESPACE}" "uaa" "INTERNAL_CA_CERT")"
 
-helm upgrade ${RELEASE} output/helm \
-    --namespace ${NAMESPACE} \
+helm upgrade "${NAMESPACE}" output/helm \
+    --name "${NAMESPACE}" \
```

Comment:
Does `helm upgrade` take a `--name`?

Reply:
Oops, no, it doesn't. Fixed in the rebased branch.